library/ix-dev/community/netbootxyz/Chart.yaml

```diff
@@ -4,9 +4,9 @@ description: netboot.xyz lets you PXE boot various operating system installers o
 annotations:
   title: netbootxyz
 type: application
-version: 1.0.7
+version: 1.0.8
 apiVersion: v2
-appVersion: 0.7.1.3
+appVersion: 0.7.3.1
```

Comment:
It would be `0.7.3.1` or `0.7.3` ?

Reply:
The version contains 2 components, and the last .x contains the build number for one of them